### PR TITLE
Going with Postgres

### PR DESCRIPTION
**Pull request by matzew**
Using Postgres, and only expose it internally (via secret). Similar to Keycloak APB. 

No need as for MYSQL user/admin ...

### CODE REVIEW

File: roles/provision-unifiedpush-apb/defaults/main.yml [hunk 1]
```diff
@@ -1,14 +1,17 @@
 # UPS Values
 ups_secret_name: "unified-push-server"
-ups_image: " docker.io/aerogear/unifiedpush-wildfly"
-ups_image_tag: "1.3.0-no-auth.Final"
+ups_image: " docker.io/matzew/unifiedpush-psql"
+ups_image_tag: "latest"
```
Comment (matzew) [inline roles/provision-unifiedpush-apb/defaults/main.yml:4]: yes, for now (see coming PRs) this is the "test" image to use .. and to avoid confusion :-)

**Comment by pb82** [comment]
👀

**Review by pb82 — APPROVED** [review]
Tested locally

**Comment by pb82** [comment]
@matzew Let's merge it :shipit:

**Review by psturc — APPROVED** [review]
Verified on singlenode cluster

**Pull request merged**